packages/server/index.js

```diff
@@ -71,6 +72,8 @@ if (cluster.isMaster) {
   mongoose.connect(config.database, {
     useNewUrlParser: true,
     useUnifiedTopology: true,
+    // Setting useFindAndModify to false ensures that Mongoose uses the native findOneAndUpdate function instead of the deprecated findAndModify function. This helps to avoid deprecation warnings and ensures that you are using the most up-to-date underlying method for these operations.
+    useFindAndModify: false,
```

Comment:
It can be risky to change mongoose options like that, have you tested everything still works and are we sure that we really need this configuration?

Reply:
Not in the scope, we don't do it please.

Reply:
Okay, I understand . For now, I'll leave the code as it is without modifying the `useFindAndModify` option. Any ways, I think we have to know that we will continue to receive the following deprecation warning:

```
(node:19600) DeprecationWarning: Mongoose: `findOneAndUpdate()` and `findOneAndDelete()` without the `useFindAndModify` option set to false are deprecated. See: https://mongoosejs.com/docs/deprecations.html#findandmodify
```